import-areas/import-areas.go

```diff
@@ -226,11 +239,25 @@ func postAreaQuery(query string) ([]models.Area, error) {
 	v.Set("query", query)
 
 	client := &http.Client{}
-	req, _ := http.NewRequest("POST", "http://statistics.data.gov.uk/sparql", strings.NewReader(v.Encode()))
+	req, err := http.NewRequest("POST", "http://statistics.data.gov.uk/sparql", strings.NewReader(v.Encode()))
+	if err != nil {
+		log.Event(ctx, "error returned from building an area request", log.ERROR, log.Error(err))
+		return nil, err
+	}
+
 	req.Header.Add("Accept", "application/sparql-results+json")
-	resp, _ := client.Do(req)
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Event(ctx, "error returned from the POST request", log.ERROR, log.Error(err))
+		return nil, err
+	}
+
```

Comment:
You also need to check the response status code - atm it assumes no error means all is well - but this could return a non `2xx` status. If `status != 200` I guess we would want to bail out and return an error.

Reply:
I have added this in

Reply:
Thank you for pointing this out, I have updated and check that the script still works